ui/component/walletExport/view.jsx

```diff
@@ -0,0 +1,75 @@
+// @flow
+import React from 'react';
+import Button from 'component/button';
+import { Form, FormField } from 'component/common/form';
+import Card from 'component/common/card';
+import Lbry from 'lbry';
+import { clipboard } from 'electron';
+import * as ICONS from 'constants/icons';
+import FileExporter from 'component/common/file-exporter';
+
+type Props = {
+  toast: (string, boolean) => void,
+};
+
+function WalletExport(props: Props) {
+  const { toast } = props;
+
+  const [password, setPassword] = React.useState();
+  const [data, setData] = React.useState();
+  const [fetching, setFetching] = React.useState();
+  const getData = async (password) => {
+    setFetching(true);
+    const data = await Lbry.sync_apply({ password });
+    setFetching(false);
+    setData(data);
+  };
+
+  return (
+    <Card
+      title={__('Export Wallet')}
+      subtitle={
+        data
+          ? __('Your Sync Data is prepared. You can now copy or save it.')
+          : __('Export wallet sync data. Choose a secure password you will need for import.')
```

Comment:
Just some thoughts -- not sure if there's any action item here.

- Since "import" is mentioned, I (as a user) was wondering how to I import?  Might open up more questions for Help Desk.
- Also wondered how to decrypt to inspect the seeds/keys for fun.

Is it possible to decode the `sync_apply` data into a file and export as-is like Desktop (minus the zip if can't do in web)?
- In this case, we don't need to care or mention about importing -- do what you want with the wallet data (e.g. keep securely, ask Help Desk to help restore, etc.)
- Let them encrypt the exported file themselves like in Desktop.

Reply:
The idea was to be able to export to other apps. Odysee has no need to import necessarily.
At the moment, only the sdk knows how to decrypt it. 
You can sync apply <password> --data=<exported> --wallet_id="somekey" I think.